esitoolsupport/__init__.py

```diff
@@ -1,97 +0,0 @@
-# Copyright 2009-2015 Eucalyptus Systems, Inc.
```

Comment:
When you remove __init__.py from a directory python stops considering that directory a module.  esitoolsupport.esibase thus cannot import.

It probably worked during your tests because an __init__.pyc file was left over.